readme.md

```diff
@@ -250,6 +250,8 @@ excluding `remark-lint-no-` or `remark-lint-`
     — ⚠️ Warn about leading white-space
 *   [`remark-lint-list-item-punctuation`](https://github.com/wemake-services/remark-lint-list-item-punctuation)
     — ⚠️ check if list items end in periods
+*   [`remark-lint-are-links-valid`](https://github.com/wemake-services/remark-lint-are-links-valid)
+    - check if your links are reachable and/or unique
```

Comment:
1. Could you use an em-dash here (`—`) instead of a hyphen-minus?
2. Could you capitalise the `C` in `check`?